docs/site/src/components/Rule.astro

```diff
@@ -0,0 +1,39 @@
r=%224%22 fill=%22%2343434396%22%2F%3E%3Ccircle cx=%22337.5%22 cy=%221363.5%22 r=%224%22 fill=%22%23434343d8%22%2F%3E%3Ccircle cx=%22418.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434313%22%2F%3E%3Ccircle cx=%22526.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434324%22%2F%3E%3Ccircle cx=%22796.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434381%22%2F%3E%3Ccircle cx=%22877.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434376%22%2F%3E%3Ccircle cx=%221147.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434365%22%2F%3E%3Ccircle cx=%221417.5%22 cy=%221363.5%22 r=%224%22 fill=%22%23434343e1%22%2F%3E%3Ccircle cx=%221471.5%22 cy=%221363.5%22 r=%224%22 fill=%22%23434343ef%22%2F%3E%3Ccircle cx=%221552.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434363%22%2F%3E%3Ccircle cx=%221822.5%22 cy=%221363.5%22 r=%224%22 fill=%22%23434343a4%22%2F%3E%3Ccircle cx=%221903.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434347%22%2F%3E%3Ccircle cx=%221984.5%22 cy=%221363.5%22 r=%224%22 fill=%22%2343434312%22%2F%3E%3Ccircle cx=%2240.5%22 cy=%221390.5%22 r=%224%22 fill=%22%234343438e%22%2F%3E%3Ccircle cx=%2267.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434382%22%2F%3E%3Ccircle cx=%22121.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343dc%22%2F%3E%3Ccircle cx=%22229.5%22 cy=%221390.5%22 r=%224%22 fill=%22%234343432d%22%2F%3E%3Ccircle cx=%22526.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434351%22%2F%3E%3Ccircle cx=%22661.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343b8%22%2F%3E%3Ccircle cx=%22688.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434317%22%2F%3E%3Ccircle cx=%22715.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343e6%22%2F%3E%3Ccircle cx=%22850.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434331%22%2F%3E%3Ccircle cx=%22958.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434370%22%2F%3E%3Ccircle cx=%221228.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343a9%22%2F%3E%3Ccircle cx=%221255.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343b6%22%2F%3E%3Ccircle cx=%221309.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434371%22%2F%3E%3Ccircle cx=%221363.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343f0%22%2F%3E%3Ccircle cx=%221444.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343e5%22%2F%3E%3Ccircle cx=%221525.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434362%22%2F%3E%3Ccircle cx=%221579.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343f8%22%2F%3E%3Ccircle cx=%221660.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434375%22%2F%3E%3Ccircle cx=%221822.5%22 cy=%221390.5%22 r=%224%22 fill=%22%234343432e%22%2F%3E%3Ccircle cx=%221849.5%22 cy=%221390.5%22 r=%224%22 fill=%22%2343434367%22%2F%3E%3Ccircle cx=%221957.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343cf%22%2F%3E%3Ccircle cx=%222011.5%22 cy=%221390.5%22 r=%224%22 fill=%22%23434343e0%22%2F%3E%3C%2Fg%3E%3C%2Fg%3E%3Cdefs%3E%3CradialGradient id=%22a%22%3E%3Cstop offset=%220%22 stop-color=%22%23fff%22 stop-opacity=%220%22%2F%3E%3Cstop offset=%221%22 stop-color=%22%23fff%22 stop-opacity=%22.6%22%2F%3E%3C%2FradialGradient%3E%3C%2Fdefs%3E%3C%2Fsvg%3E");
```

Comment:
this url is meant to be like this?

Reply:
Yes, it gives the background a cool pattern